### PR TITLE
doc(release): Adding 2.8.28 release note

### PR DESCRIPTION
**Pull request by adr-mo**
# Pull Request Template

## Description

Release note added for 2.8.28

**Fixes** None

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [x] Updating documentation (missing information, typo...)

## Target serie

- [x] 2.8.x
- [ ] 18.10.x
- [ ] 19.04.x
- [ ] 19.10.x (master)

<h2> How this pull request can be tested ? </h2>

Display the release_note

#### Community contributors & Centreon team

- [ ] I followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

#### Centreon team only

- [ ] I have made sure that the **unit tests** related to the story are successful.
- [ ] I have made sure that **unit tests cover 80%** of the code written for the story.
- [ ] I have made sure that **acceptance tests** related to the story are successful (**local and CI**)

### CODE REVIEW

File: doc/fr/release_notes/centreon-2.8/centreon-2.8.28.rst [hunk 1]
```diff
@@ -1,24 +1,24 @@
-###################
-Centreon Web 2.8.28
-###################
+  1 ###################
```
Comment (ganoze) [inline doc/fr/release_notes/centreon-2.8/centreon-2.8.28.rst]: Wrong content I guess.

**Comment by adr-mo** [comment]
I've based the release note on the JIRA tickets marked as **Resolved**.
I guess I'll recheck regarding what Laurent has added previously

**Comment by ganoze** [comment]
Sorry I misexplained. Each line starting from the one I mentioned have a number before the real content. I suppose that this is a diff copy-paste error.

**Comment by adr-mo** [comment]
Ow Ok, I've pushed a new commit for it then :)

Thanks

File: doc/fr/release_notes/centreon-2.8/centreon-2.8.28.rst (hunk 1 above)
Comment (adr-mo) [reply doc/fr/release_notes/centreon-2.8/centreon-2.8.28.rst]: **Fixed**

**Pull request merged**